test/integration/features/proxy.feature

```diff
@@ -0,0 +1,87 @@
+@proxy @minishift-only
+Feature: Minishift can run behind proxy
+  As a user I can use Minishift behind a proxy.
+
+  Scenario: Start behind the proxy
+   Given user starts proxy server and sets MINISHIFT_HTTP_PROXY variable
```

Comment:
Or maybe you don't need to set anything at all? I see that setProxt sets the proxy variable.